### PR TITLE
participate.html: mention the Matrix rooms / XMPP bridge

### PR DESCRIPTION
**Pull request by Mikaela**
Resolves: #1674

* Netlify preview for the mainly edited page: https://deploy-preview-1775--privacytools-io.netlify.com/index.html#participate

### CODE REVIEW

File: _includes/sections/participate.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+<p>It's important for a website like {{ site.name }} to stay up-to-date. Keep an eye on software updates for the applications listed on our site. Follow recent news about providers that we recommend. We try our best to keep up, but we're not perfect and the internet is changing fast. If you find an error, or you think a provider should not be listed here, or a qualified service provider is missing, or a browser plugin is not the best choice anymore, or anything else... <strong>Talk to us please.</strong> You can also find us on <a rel="me" href="https://social.privacytools.io/@privacytools">our own Mastodon instance</a> or on <a href="https://chat.privacytools.io">several Matrix rooms</a> mainly <code class="highlighter-rouge">#general:privacytools.io</code>. XMPP users can join there through <a href="https://conversations.im/j/%23general%23privacytools.io@matrix.org">#general#privacytools.io@matrix.org</a>.</p>
```
Comment (Mikaela) [inline _includes/sections/participate.html]: > Talk to us please. You can also find us on our own Mastodon instance or on several Matrix rooms mainly #general:privacytools.io. XMPP users can join there through #general#privacytools.io@matrix.org.

I am not entirely happy with this by myself either, but the Matrix rooms have expanded from just that one and I don't want to start removing general without consulting anyone and I guess it's still main/hub room where people can be sent to correct direction if they want to delve deeper into the wiki or anything.

Comment (blacklight447-ptio) [reply _includes/sections/participate.html]: its fine with me, it makes for a nice landing spot for new folks

**Comment by Mikaela** [comment]
TODO: fix merge conflict

**Comment by Mikaela** [comment]
@JonahAragon I think you are needed to poke Netlify here

**Comment by Mikaela** [comment]
Assigning @JonahAragon for Netlify poking.

**Comment by Mikaela** [comment]
So `git commit --allow-empty -m "whatever"` lets you poke Netlify without losing previous approving reviews.

**Pull request merged**